db/migrate/20171031131806_remove_publication.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePublication < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :books, :publication
```

Comment:
this migration is irreversible. Write this here `remove_column :books, :publication, :string`